docs/assets/dev/js/editor/document/history/readme.md

```diff
@@ -0,0 +1,145 @@
+| [Undo](#command----erundocumenthistroyundo)                            | `$e.run('document/history/undo')`                  | Undo history item.
+| [Undo-All](#command----erundocumenthistroyundo-all)                    | `$e.run('document/history/undo-all')`              | Undo all history items.
+| [Redo](#command----erundocumenthistroyredo)                            | `$e.run('document/history/redo')`                  | Redo history item.
+
+
+## Component `document/history/` -- Internal Commands
+| Command                                                                     | Access                                              | Description
+|-----------------------------------------------------------------------------|-----------------------------------------------------|-----------------------------------------
+| [Add-Transaction](#command----einternaldocumenthistroyadd-transaction)      | `$e.internal('document/history/add-transaction')`   | Add transaction item.
+| [Clear-Transaction](#command----einternaldocumenthistroyclear-transaction)  | `$e.internal('document/history/clear-transaction')` | Clear transaction.
+| [Delete-Log](#command----einternaldocumenthistroydelete-log)                | `$e.internal('document/history/delete-log')`        | Delete log.
+| [End-Log](#command----einternaldocumenthistroyend-log)                      | `$e.internal('document/history/end-log')`           | End log.
+| [End-Transaction](#command----einternaldocumenthistroyend-transaction)      | `$e.internal('document/history/end-transaction')`   | End Transaction.
+| [Log-Sub-Item](#command----einternaldocumenthistroylog-sub-item)            | `$e.internal('document/history/log-sub-item')`      | Lob sub item.
+| [Start-Log](#command----einternaldocumenthistroystart-log)                  | `$e.internal('document/history/start-log')`         | Start log.
+
+## _Command_ -- `$e.run('document/histroy/do')`
+*  **Name**: Do.
+*  **Description**: Do history step.
+*  **Returns**: `{void}`
+
+   | Property     | Type                  | Requirement    | Description |
+   |---           |---                    |---             |---|
+   | _index_         | `{Number}`         | **optional**   | Index of history item.
+
+## _Command_ -- `$e.run('document/histroy/undo')`
+*  **Name**: Undo.
+*  **Description**: Undo history step.
+*  **Returns**: `{void}`
+*  **Arguments**: None.
+
+## _Command_ -- `$e.run('document/histroy/undo-all')`
+*  **Name**: Undo-All.
+*  **Description**: Undo all history step.
+*  **Returns**: `{void}`
+*  **Arguments**: None.
+
+## _Command_ -- `$e.run('document/histroy/redo')`
+*  **Name**: Redo.
+*  **Description**: Redo history step.
+*  **Returns**: `{void}`
+*  **Arguments**: None.
+
+## _Command_ -- `$e.internal('document/histroy/add-transaction')`
+*  **Name**: Add-Transaction.
+*  **Description**: Add item to transactions.
```

Comment:
Maybe explain a little bit about the differences between items, logs and transactions?

Reply:
Handled.